### DISCUSSION TITLE
Vertical alignment of specific text

### DISCUSSION THREAD

**Discussion by bpffjl** [davidgohel/officedown#126] [Q&A]
I'm working on some reports where the first page needs to be vertically aligned to the center, but all other pages are vertically aligned to the top. The reports are generated with Rmarkdown, using officerdown and a specific word template. Are there functions in the officerverse that can change vertical alignment of a selected text?

**Comment by MeganMcAuliffe** [comment]
I would like to know if you figured this out as well. My fp_text always center vertically aligns when i want it at the top.

**Comment by davidgohel** [comment]
I don't think vertical alignment is a text or paragraph option in Word (not sure but almost!). To me you want `padding` in `fp_par()`.
